content/news/2021-11-galaxy-release-21-09/index.md

```diff
@@ -0,0 +1,89 @@
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/vCapB1xHjZg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Unfortunately not all tools are well annotated yet, and while we also pull terms from [bio.tools](https://bio.tools/), this still doesn't get us full coverage and many tools will still appear under a large section "Uncategorized". Hopefully this will improve as users and tool developers annotate these tools with their terms to help everyone find them.
+
+### Collections (Beta History Updates)
+
+Did you ever create *a collection with the wrong dbkey or datatype*? Well in the Beta History you can now **change collections' datatypes and dbkeys**! This should save a lot of time for everyone working with large datasets ([Pull Request 11799](https://github.com/galaxyproject/galaxy/pull/11799/)).
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/aPgMcTuzEJ4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Additionally, did a *collection ever fail* for you? And you wondered why? Now you can find out with the **view details button for collections** ([Pull Request 12261](https://github.com/galaxyproject/galaxy/pull/12261))!
+
+### Importing Data
+
+Selecting datasets from the Remote Files view has gotten easier! Now you can select folders and files, and it will import all of the datasets recursively in those folders. Previously you could only select files within a folder so this is a huge improvement in usability of such a key new feature of Galaxy ([Pull Request 12310](https://github.com/galaxyproject/galaxy/pull/12310)). But with great power comes great responsibility, be careful not to import the entirety of NCBI!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/XcZgQbGojh8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Reports
+
+Report components used to try and arrange themselves in a smart way, but due to some issues with the floating of report components these will now be full width until we can figure out a good way to provide a similar feature allowing you to control the layout of the report ([Pull Request 12085](https://github.com/galaxyproject/galaxy/pull/12085)).
```

Comment:
```suggestion
Report components used to automatically arrange themselves in a smart way, but due to some issues with the floating of components, these will now be full width until we can figure out a good way to provide a similar feature, allowing you to control the layout of the report ([Pull Request 12085](https://github.com/galaxyproject/galaxy/pull/12085)).
```

Reply:
`allowing you to control the layout of the report` is a bit confusing to me: Does it mean that in the future users will be able to arrange the components or is it happening already?

Reply:
In the future, I think we can just drop that comment.